src/otx/core/model/entity/instance_segmentation.py

```diff
@@ -131,7 +149,10 @@ def _customize_outputs(
                     canvas_size=output.img_shape,
                 ),
             )
-            output_masks = tv_tensors.Mask(output.pred_instances.masks, dtype=torch.bool)
+            output_masks = tv_tensors.Mask(
+                output.pred_instances.masks,
+                dtype=torch.bool,
+            )
```

Comment:
Is this change come from ruff-formatter?

Reply:
No but ruff-formatter allowed it